core/dbt/config/project.py

```diff
@@ -137,11 +137,10 @@ def _all_source_paths(
     analysis_paths: List[str],
     macro_paths: List[str],
 ) -> List[str]:
-    # We need to turn a list of lists into just a list, then convert to a set to
-    # get only unique elements, then back to a list
-    return list(
-        set(list(chain(model_paths, seed_paths, snapshot_paths, analysis_paths, macro_paths)))
-    )
+    paths = chain(model_paths, seed_paths, snapshot_paths, analysis_paths, macro_paths)
+    # Strip trailing slashes since the path is the same even though the name is not
+    paths = map(lambda s: s.rstrip("/"), paths)
+    return list(set(paths))
```

Comment:
my doesn't like that you change the type of `paths` here
```suggestion
    paths_stripped = map(lambda s: s.rstrip("/"), paths)
    return list(set(paths_stripped))
```

Reply:
I did this before, with the constant updates to the underlying code I must've messed up the rebase. Let me fix that